src/Ydb.Protos/Codegen/Program.cs

```diff
@@ -0,0 +1,56 @@
+using System.Text;
```

Comment:
Need move the generator helper from src/Ydb.Protos

Ydb.Protos - is will be generated code only